pyogrio/_io.pyx

```diff
@@ -757,7 +861,12 @@ def ogr_read_info(str path, object layer=None, object encoding=None, **kwargs):
         'encoding': encoding,
         'fields': get_fields(ogr_layer, encoding)[:,2], # return only names
         'geometry_type': get_geometry_type(ogr_layer),
-        'features': OGR_L_GetFeatureCount(ogr_layer, 1)
+        'features': OGR_L_GetFeatureCount(ogr_layer, 1),
+        "capabilities": {
+            "random_read": OGR_L_TestCapability(ogr_layer, OLCRandomRead),
+            "fast_set_next_by_index": OGR_L_TestCapability(ogr_layer, OLCFastSetNextByIndex),
+            "fast_spatial_filter": OGR_L_TestCapability(ogr_layer, OLCFastSpatialFilter),
```

Comment:
Do we want to keep this?

Reply:
This seems potentially useful, at least for benchmarking.  I'd keep it for now.